test/unit/host_test.rb

```diff
@@ -134,11 +134,20 @@ class HostTest < ActiveSupport::TestCase
   end
 
   test "sets compute attributes on create" do
-    Host.any_instance.expects(:set_compute_attributes).once.returns(true)
```

Comment:
I don't think this test should be changed, it's here to test the before_validation step.